Makefile

```diff
@@ -25,21 +25,25 @@ IS_WIN  := $(strip $(if $(or $(IS_LINUX),$(IS_OSX),$(IS_WSL)),,$(OS)))
 $(if $(IS_WIN),$(error Windows is not supported in all recipes. Use WSL instead. Follow instructions in README.md),)
 
 # VARIABLES ----------------------------------------------
-# TODO: read from docker-compose file instead $(shell find  $(CURDIR)/services -type f -name 'Dockerfile')
-# or $(notdir $(subst /Dockerfile,,$(wildcard services/*/Dockerfile))) ...
-SERVICES_LIST := \
-	api-server \
-	autoscaling \
-	catalog \
-	dask-sidecar \
-	datcore-adapter \
-	director \
-	director-v2 \
-	dynamic-sidecar \
-	migration \
-	static-webserver \
-	storage \
-	webserver
+# NOTE: Name given to any of the services that must be build, regardless
+# whether they are part of the simcore stack or not. This list can be obtained from
+#
+# cat services/docker-compose-build.yml | yq ".services | keys | sort"
```

Comment:
Question: why not generate it like this instead of having it statically defined?

Reply:
Second question: the only change was adding the missing service to this list, right?

Reply:
> * because `yq` is not part of our official toolset (hopefully soon).

Maybe yq can be used via a docker image https://hub.docker.com/r/mikefarah/yq